spec/lib/identity_job_log_subscriber_spec.rb

```diff
@@ -29,4 +29,30 @@
       issuer: build(:service_provider).issuer,
     )
   end
+
+  it 'logs errors when exception occurs in job' do
+    expect(Rails.logger).to receive(:error).exactly(2).times do |log|
+      json = JSON.parse(log)
+
+      expect(json['name']).to be_in [
+        'enqueue.active_job', 'perform.active_job'
+      ]
+      expect(json['job_class']).to eq('AddressProofingJob')
+      expect(json.key?('trace_id'))
+      expect(json.key?('duration_ms'))
+      expect(json.key?('job_id'))
+      expect(json.key?('timestamp'))
+      expect(json['exception_message']).to eq 'invalid base64'
+    end
+
+    expect {
+      AddressProofingJob.perform_later(
+        result_id: nil,
+        encrypted_arguments: 'abc',
+        trace_id: nil,
+        user_id: SecureRandom.random_number(1000),
+        issuer: build(:service_provider).issuer,
+      )
+    }.to raise_error(ArgumentError)
```

Comment:
lint, this will need do..end right?

```suggestion
    expect do
      AddressProofingJob.perform_later(
        result_id: nil,
        encrypted_arguments: 'abc',
        trace_id: nil,
        user_id: SecureRandom.random_number(1000),
        issuer: build(:service_provider).issuer,
      )
    end.to raise_error(ArgumentError)
```

Reply:
Either work, but I do think we usually use do/end